src/chrome/content/rules/hkpl.gov.hk.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+subdomains:
+	- portal-216.	(cert mismatch www.hkpl.gov.hk)
+	- hkclweb.	(cert mismatch mmis.hkpl.gov.hk)
+-->
+
+<ruleset name="Hong Kong Public Libraries">
+	<target host="hkpl.gov.hk" />
+	<target host="www.hkpl.gov.hk" />
+	<target host="mmis.hkpl.gov.hk" />
+
+	<rule from="^http(s?)://hkpl\.gov\.hk/"
```

Comment:
Please document the mismatch.

Is there any case, you can get on the https version without explicit typing it? Otherwise we should just redirect http.